smack-openpgp/build.gradle

```diff
@@ -8,7 +8,9 @@ dependencies {
 	compile project(':smack-extensions')
 	compile project(':smack-experimental')
 
-	compile 'org.pgpainless:pgpainless-core:0.0.1-alpha7'
+	api 'org.pgpainless:pgpainless-core:0.0.1-alpha9'
+
+	implementation "org.bouncycastle:bcprov-jdk15on:1.65"
```

Comment:
I'd like to keep bc out of the dependencies (if possible). I assume this is required for the unit tests? If so s/implementation/testImplementation/

Reply:
Done :)